src/main/java/core/basesyntax/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        boolean isFirstLine = true;
+        for (String line : data) {
+            if (isFirstLine) {
+                isFirstLine = false;
+                continue;
+            }
+            String[] parts = line.split(",");
+            if (parts.length != 3) {
+                continue;
+            }
+            FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
+            transactions.add(new FruitTransaction(operation, fruit, quantity));
+        }
+        return transactions;
+    }
```

Comment:
The method `convertToTransaction` should not silently skip incorrect data. It would be better to throw an exception if the data format is not as expected, as this indicates an issue with the input file that should be addressed.